src/tools/illink/test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresInRootAllAssembly.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.RequiresCapability
+{
+	[SetupLinkerArgument ("-a", "test.exe")]
```

Comment:
Doesn't `-a test.exe` without a mode just root the entry point, not the entire assembly?

```suggestion
	[SetupLinkerArgument ("-a", "test.exe", "all")]
```